website/sidebars.js

```diff
@@ -152,34 +152,41 @@ module.exports = {
       type: "category",
       label: "Introduction",
       collapsible: true,
-      items: ["introduction/what_is", "introduction/vs_old"],
+      items: ["v0.1/introduction/what_is", "v0.1/introduction/vs_old"],
     },
     {
       type: "doc",
-      id: "install",
+      id: "v0.1/install",
     },
     {
       type: "category",
       label: "Learn Dagger",
       collapsible: true,
       collapsed: false,
       items: [
-        "learn/what_is_cue",
-        "learn/get-started",
-        "learn/google-cloud-run",
-        "learn/kubernetes",
-        "learn/aws-cloudformation",
-        "learn/github-actions",
-        "learn/dev-cue-package",
-        "learn/package-manager",
+        "v0.1/learn/what_is_cue",
+        "v0.1/learn/get-started",
+        "v0.1/learn/google-cloud-run",
+        "v0.1/learn/kubernetes",
+        "v0.1/learn/aws-cloudformation",
+        "v0.1/learn/github-actions",
+        "v0.1/learn/dev-cue-package",
+        "v0.1/learn/package-manager",
       ],
     },
     {
       type: "category",
       label: "Use Cases",
       collapsible: true,
       collapsed: true,
-      items: ["use-cases/ci"],
+      items: ["v0.1/use-cases/ci"],
+    },
+    {
+      type: "category",
+      label: "Guidelines",
```

Comment:
Note this section is not part of the current 0.1 docs. I have added it to verify that common files can be shared between versions.